packages/design-system/src/components/table/components/index.tsx

```diff
@@ -95,6 +100,14 @@ const Table = ({
     [showColumnsMenu]
   );
 
+  const exportCookies = useCallback(() => {
+    const cookies = table.rows.map(({ originalData }) => originalData);
+    if (cookies.length > 0 && 'parsedCookie' in cookies[0]) {
+      const csvTextBlob = generateCookieTableCSV(cookies as CookieTableData[]);
+      saveAs(csvTextBlob, 'out.csv');
```

Comment:
`out.csv` seems a bit vague. 
Can we use another name for the output file, making it more inclusive?
Maybe `tableReport` or something, I'm open to others too.

Reply:
Rename to "Cookies Report.csv"